_docs/quickstarts/scanner/rules.md

```diff
@@ -67,6 +69,8 @@ rules:
 
 ## Kubernetes Engine (KE) rules
```

Comment:
Please don't abbreviate Kubernetes Engine when referring to the Google product. We should always use the full product name. Thanks!

Reply:
Done